docker-bake.hcl

```diff
@@ -1,11 +1,18 @@
 target "aws" {
     target = "aws"
-    platforms = ["linux/amd64", "linux/arm64"]
+    platforms = ["linux/amd64"]
```

Comment:
?

Reply:
Forget to add it back after testing locally for amd64 only, sorry